game_frontend/src/babylon/assetPacks/ancient.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  StandardMaterial,
+  Texture,
+  Color3,
+  Vector3,
+  TransformNode,
+  AbstractMesh,
+  Scene
+} from 'babylonjs'
+import AssetPack from './assetPack'
+
+export default class AncientAssetPack extends AssetPack {
+    obstacleMaterial: StandardMaterial
+
+    constructor (era: string, scene: Scene) {
+      super(era, scene)
+      this.obstacleMaterial = this.makeObstacleMaterial()
+    }
+
+    makeObstacleMaterial (): StandardMaterial {
+      const material = new StandardMaterial(this.obstacleInfo.materialName, this.scene)
+      material.diffuseTexture = new Texture(this.obstacleInfo.textureURL, this.scene)
+      material.specularColor = new Color3(0, 0, 0)
+      material.diffuseColor = new Color3(1, 1, 1)
+      return material
+    }
+
+    async createObstacle (
+      name: string,
+      location: Vector3,
+      parent: TransformNode
+    ): Promise<AbstractMesh> {
+    // TODO: There are two different models for the same texture, but only one is used in here
```

Comment:
I would remove the TODO here and make this a normal comment or find another place for this. (perhaps some documentation on the AssetPacks?)